src/core/toga/app.py

```diff
@@ -47,6 +47,8 @@ class App:
         >>> app.main_loop()
     """
     app = None
+    app_dir = None
+    app_module = None
```

Comment:
Why are these two needed? You should be able to confirm if `app_dir` has a value by checking if `app` exists

Reply:
I was getting an error with the "App.app_dir".  It said App has no attribute "app_dir". I wasn't sure why the App.app_dir was doing since app_dir was only available when a new app is created.